.github/workflows/srtool-build.yml

```diff
@@ -0,0 +1,95 @@
+name: srtool build
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  SUBWASM_VERSION: 0.20.0
+  TOML_CLI_VERSION: 0.2.4
+
+jobs:
+  build-wasm:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        runtime: [peregrine, spiritnet]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache build and output directories
```

Comment:
For the WASM builds I think it's better not to use any previous cache.

Reply:
Also, we can afford to wait a bit longer, since this process won't happen that often.